TabnineSelfHosted/src/main/java/com/tabnineSelfHosted/statusBar/SelfHostedStatusBarActions.kt

```diff
@@ -10,27 +11,34 @@ import com.tabnineCommon.binary.requests.login.LoginRequest
 import com.tabnineCommon.binary.requests.login.LogoutRequest
 import com.tabnineCommon.general.DependencyContainer
 import com.tabnineCommon.userSettings.AppSettingsConfigurable
+import com.tabnineCommon.userSettings.AppSettingsState
 
 const val OPEN_TABNINE_SETTINGS_TEXT = "Open Tabnine settings"
 const val LOGIN_TEXT = "Sign in to Tabnine"
 const val LOGOUT_TEXT = "Sign out of Tabnine"
+const val GOTO_FAQ_TEXT = "Get help"
+const val FAQ_URL = "https://support.tabnine.com/hc/en-us/articles/5760725346193-Connectivity-possible-issues"
 
 object SelfHostedStatusBarActions {
     private val binaryRequestFacade = DependencyContainer.instanceOfBinaryRequestFacade()
 
     @JvmStatic
     fun buildStatusBarActionsGroup(
         project: Project?,
-        isLoggedIn: Boolean
+        isLoggedIn: Boolean?,
     ): DefaultActionGroup {
         val actions = ArrayList<AnAction>()
-        actions.add(
-            if (isLoggedIn) {
-                createLogoutAction()
-            } else {
-                createLoginAction()
-            }
-        )
+        if (AppSettingsState.instance.cloud2Url.isNotBlank()) {
+            actions.add(
+                if (isLoggedIn == null) {
+                    createGoToFAQAction()
+                } else if (isLoggedIn) {
+                    createLogoutAction()
+                } else {
+                    createLoginAction()
+                }
```

Comment:
i don't like this usage of `null`s - i'd create an enum `UserLoginStatus` with 3 options: `LoggedIn`, `LoggedOut`, and `NotAvailable` or something.